### DISCUSSION TITLE
Trigger for Daily Consumption Data Updates

### DISCUSSION THREAD

**Discussion by sentur66413** [BottlecapDave/HomeAssistant-OctopusEnergy#563] [Q&A] [answered]
Is there an event or some sort of trigger for when previous days consumption data is updated? 

Referencing the docs I can’t see an event. 

I’d like to use an automation when the data and cost of the previous days usage comes in. So I can get a notification. 

I find the data gets updated around 4pm most days. But it’s not super reliable. Sometimes it’s later or not at all if there’s been an issue with Octopus retrieving the smart metre data.

**Comment by BottlecapDave** [comment, marked as answer]
Hello. You can trigger on the `last_evaluated` attribute on the cost and consumption sensors. When these values change, it means the state has been updated.
